RELEASES.md

```diff
@@ -18,7 +18,7 @@
   implication that building *without* this feature makes this a `#![no_std]`
   crate.  A few methods now require `FloatCore` instead of `Float`.
 - [The `serde` dependency has been updated to 1.0][24], and `rustc-serialize`
-  is no longer supported by `num-complex`.
+  is no longer supported by `num-bigint`.
```

Comment:
Yes, this is a typo, but the correct fix is to reference *this* crate, `num-rational`.  (Or just drop the "by $crate" altogether.)